### PR TITLE
Wrap game library button from titlebar games plugin into a div

### PR DESCRIPTION
**Pull request by gear4s**

### CODE REVIEW

File: src/Powercord/plugins/pc-titleBarGames/style.scss [hunk 1]
```diff
@@ -16,6 +16,8 @@
     transition: 0.5s;
     cursor: pointer;
     border-radius: 2px;
+    margin-right: 0;
+	background-position: center center;
```
Comment (cyyynthia) [inline src/Powercord/plugins/pc-titleBarGames/style.scss]: this is not indented properly

File: src/Powercord/plugins/pc-titleBarGames/index.js [hunk 2]
```diff
@@ -45,11 +45,12 @@ module.exports = class TitleBarGames extends Plugin {
...
       );
 
-    applications.push(React.createElement(this.icon, {
-      name: 'Library',
-      className: `pc-game-img ${this.iconClass.linkButtonIcon}`,
-      onClick: () => this.navigator.transitionTo(Routes.APPLICATION_LIBRARY)
-    }));
+    applications.push(
+      React.createElement('div', {
+        className: `pc-game-img ${this.iconClass.linkButtonIcon}`,
```
Comment (cyyynthia) [inline src/Powercord/plugins/pc-titleBarGames/index.js]: As I said on a previous PR, `.pc-` is usually reserved to normalized classes and for custom ones plugins usually uses `.powercord-`. If you can update it in this PR that'd be awesome :)

Comment (gear4s) [reply src/Powercord/plugins/pc-titleBarGames/index.js]: this is from your plugin but i can update if you wish

Comment (davidcralph) [reply src/Powercord/plugins/pc-titleBarGames/index.js]: 200iq

Comment (gear4s) [reply src/Powercord/plugins/pc-titleBarGames/index.js]: done, can't test as it titleBarGames uh .. crashes my discord. plz fix

**Review by cyyynthia — APPROVED** [review]
Looks good to me, i'll merge it even if we can't really test it. We'll debug it if needed once the plugin itself will work again lol

**Pull request merged**